### ISSUE TITLE
How to run peak calling only?

### ISSUE THREAD

**Issue by BavenBijuMathews** [mnsmar/clipseqtools#10]
I have computed alignments of deduplicated reads already and I want to input them for peak-calling. I am not sure how to best do this. What format is .db for database tag in 'clipseqtools' application? Is there a way to use bam/bed files as input?

clipseqtools all \
  --database proteinA/reads.adtrim.star_Aligned.out.single.sorted.collapsed.db \
  --gtf data/hg19/annotation/UCSC_gene_parts_genename.gtf \
  --rname_sizes data/hg19/chrom.sizes \
  --o_prefix clip/proteinA/ \
  --plot \
  -v

**Comment by mnsmar** [comment]
Assuming that you have the .db file from a previous step you can use

```
clipseqtools cluster_size_and_score_distribution
  --database proteinA/reads.adtrim.star_Aligned.out.single.sorted.collapsed.db
  --o_prefix clip/proteinA/
  --plot
  -v
```

**Comment by BavenBijuMathews** [comment]
How do you get to the .db file? Is there a utility tool for converting bed/bam files to that? (I have deduplicated alignments in bam format)

**Comment by mnsmar** [comment]
`clipseqtools-preprocess sam_to_sqlite`.

fyi if you just execute each of the `clipseqtools-preprocess`, `clipseqtools` and `clipseqtools-compare` it will list the available commands and a short description.

**Comment by BavenBijuMathews** [comment]
> Assuming that you have the .db file from a previous step you can use
> 
> ```
> clipseqtools cluster_size_and_score_distribution
>   --database proteinA/reads.adtrim.star_Aligned.out.single.sorted.collapsed.db
>   --o_prefix clip/proteinA/
>   --plot
>   -v
> ```

I was able to use this command successfully but the output given were cluster_score_distribution and cluster_size_distribution files. In terms of peak-calling, I was wondering if there was any other subcommand that gave output as a list of peaks in bed/bam/sam files. I went through the list of available commands for 'clipseqtools' but I wasnt able to see an option like that.

**Comment by mnsmar** [comment]
I released a new version 1.0.0. In this new version, the above command will also output a BED file with the clusters coordinates. Could you please update to the latest version from CPAN and test?

**Comment by BavenBijuMathews** [comment]
Sorry for the late reply. Yes, it outputs the needed files. Thank you so much for your help.
